avocado/utils/crypto.py

```diff
@@ -16,6 +16,8 @@
 import logging
 import hashlib
 
+def  algorithms_supported():
+    return ['md5', 'sha1', 'sha256', 'sha384', 'sha512']
```

Comment:
Well, I'm not sure why we hardcoded to accept only `md5` and `sha1`, but if we are going to expand this list, we better don't hardcode at all. We should check `hashlib.algorithms_available` then (unless you have a good reason to not include all the hashlib available algorithms).